### PR TITLE
😯 Add support for the new secret from_bytes extension.

### PR DESCRIPTION
**Pull request by acw**

### CODE REVIEW

File: test-fixtures/src/bin/secret-store.rs [hunk 1]
```diff
@@ -20,4 +21,9 @@ fn main() {
         None => {}
         _ => panic!(),
     }
+
+    // FIXME: uncomment this when we release this API more fully
+    //let hello_bytes = "hello, wasm_world!".as_bytes().to_vec();
+    //let secret = Secret::from_bytes(hello_bytes).unwrap();
+    //assert_eq!("hello, wasm_world!", secret.plaintext());
```
Comment (acfoltzer) [inline test-fixtures/src/bin/secret-store.rs:28]: Time to FIXME?

Comment (acw) [reply test-fixtures/src/bin/secret-store.rs:28]: As soon as the 0.9.5 release goes out, I'll update it :smile:

Comment (acfoltzer) [reply test-fixtures/src/bin/secret-store.rs:28]: Oh, we have ourselves a circular dependency here, don't we

**Review by acfoltzer — APPROVED** [review]

File: test-fixtures/src/bin/secret-store.rs (hunk 1 above)
Comment (acw) [reply test-fixtures/src/bin/secret-store.rs:28]: :(

Comment (acw) [reply test-fixtures/src/bin/secret-store.rs:28]: Yeah, we can either merge this as is, with a commented-out test case (I can say, I ran it manually and it worked), or we can merge a broken test case and have the SDK release fix it. I think I'd lean towards the former, and then update Viceroy afterwards.

**Review by JakeChampion — APPROVED** [review]

**Pull request merged**